.github/workflows/pre-commit.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     paths:
       - datasets/*/dataset.json
+      - datasets/*/*/*.json
```

Comment:
This doesn't solve the issue I raised, but maybe that's more of a thing to address in the validation itself.

The issue being that if a table JSON file changes the valdiation should not be run against the changed table JSON file, but against the dataset it is part of.

In addition, if I read the [spec](https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#table-refs) correctly, the table JSON files don't need to be in one directly below the directory that contains `dataset.json`, they could in a directories many levels below that.

Reply:
> Does the paths spec allow for `datasets/*/**/*.json` (mind the double `**`)?

[Yes](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)

I suppose the workflow should still trigger on push where table-splitted files have been changed. However, in that case the `CHANGED_DATASETS` should be cleaned up to remove the table-splitted files, but only point to the `dataset.json` that it's part of. Lets see if I can bash this ...